advanced_source/usb_semisup_learn.py

```diff
@@ -77,6 +77,10 @@
 # - ``Trainer``: a Trainer class for training and evaluating the
 # algorithm on dataset
 # 
+# Note that a CUDA-enabled backend is required for training with the ``semilearn`` package.
+# See [Using CUDA](https://pytorch.org/tutorials/beginner/colab#using-cuda) for instructions
```

Comment:
Technically this link doesn't exist yet... Is there a way to add a relative link to that page?